--- gnome-initial-setup-3.16.3/gnome-initial-setup/pages/account/um-realm-manager.c.orig	2014-09-02 05:37:52.000000000 +0200
+++ gnome-initial-setup-3.16.3/gnome-initial-setup/pages/account/um-realm-manager.c	2015-05-25 17:51:05.333889535 +0200
@@ -23,7 +23,7 @@
 
 #include "um-realm-manager.h"
 
-#include <krb5/krb5.h>
+#include <krb5.h>
 
 #include <glib.h>
 #include <glib/gi18n.h>
@@ -706,8 +706,10 @@
         code = krb5_get_init_creds_opt_alloc (k5, &opts);
         g_return_val_if_fail (code == 0, code);
 
+#if 0
         code = krb5_get_init_creds_opt_set_out_ccache (k5, opts, ccache);
         g_return_val_if_fail (code == 0, code);
+#endif
 
         code = krb5_get_init_creds_password (k5, &creds, principal,
                                              (char *)password,
